docs/make.jl

```diff
@@ -36,7 +36,7 @@ folder = deploy_decision.subfolder
 println("Deploying to $folder")
 vitepress_config_file = joinpath(@__DIR__, "build", ".vitepress", "config.mts")
 config = read(vitepress_config_file, String)
-new_config = replace(config, "base: 'REPLACE_ME_WITH_DOCUMENTER_VITEPRESS_BASE_URL_WITH_TRAILING_SLASH'" => "base: '/DimensionalData.jl/$folder/'")
+new_config = replace(config, "base: 'REPLACE_ME_WITH_DOCUMENTER_VITEPRESS_BASE_URL_WITH_TRAILING_SLASH'" => "base: '/DimensionalData.jl/$folder'")
```

Comment:
Hey, any particular reason for this change?  The [Vitepress docs](https://vitepress.dev/reference/site-config#:~:text=It%20should%20always%20start%20and%20end%20with%20a%20slash.) specify that the path should always end with a slash...

Reply:
Oh I was just getting some path problems in local env, I forgit I changed it